### ISSUE TITLE
Setting binary focalpoint not documented

### ISSUE THREAD

**Issue by pschulzk** [gentics/mesh#693]
Related to [MeshUI GitHub issue #166](https://github.com/gentics/mesh-ui/issues/166)

## Version
Mesh 0.28.2

## Problem
I want to set/change a binary's focalpoint properties.
However, the exact request formatting is not documented neither at https://getmesh.io/docs/api/#project__nodes__nodeUuid__binary__fieldName__post nor at https://getmesh.io/docs/features/#imagemanipulation .

My initial request contained a malformed payload and Mesh didn't respond with an error message.
But maybe that is intended behaviour.

#### Endpoint
```/api/v1/demo/nodes/{{uuid}}/binaryTransform/image```

#### Request Payload

```
{
   ...
   "focalPointX":0.97255,
   "focalPointY":0.04873,
   "cropRect":{
      ...
   }
}
```

And no ```Bad Request``` response.

#### Endpoint
```/api/v1/demo/nodes/{{uuid}}?lang={{lang}}```

#### Request Payload

```
{
   "fields":{
      "name":"Embraer Legacy 600 Image",
      "image":{
        ...
         "focalPoint":{
            "x":0.9181,
            "y":0.10025
         },
         ...
      }
   },
    ...
}
```

**Comment by Jotschi** [comment]
I'm going to fix approach I with #752  - I also added the focal point parameter to binary update request examples. I'll also add examples to the docs.

**Comment by Jotschi** [comment]
Examples have been added and will be published with next release. Thanks for reporting.
